### PR TITLE
Urban water management

### PR DESCRIPTION
**Pull request by Caroline-nc**
Groves (2008) urban water management case study example using RDM, scenario discovery and PRIM

### CODE REVIEW

File: _examples/groves_2008.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    <p> Four workshops were organised, the first three designing the simulation model to evaluate the IEUA's water management plans over a wide range of futures, and the fourth  raising decision maker awareness about the significance of climate change risks for the actual plan relative to other alternatives and evaluating means to reduce the UWMP vulnerabilities to multiple plausible futures.</p>
+    <p> The project aimed to build decision makers' capacity to plan in the long-term and adapt to changing conditions. 
+    </p>
+    <p> The UWMP was designed based on a standard probability analysis approach to identify the best management option in terms of <a href="../../theory/reliability/">reliability</a> (i.e., chance of a water shortage under 7% over the next 25 years). The RDM approach identified two major vulnerable conditions for the plans:<ul>
+        <li>a Dry, Flashy, Low-Recycling scenario (in terms of groundwater recharge): the current plan fails to prevent water shortages and imposes significant socioeconomic costs to water users.</li>
```
Comment (josephguillaume) [inline _examples/groves_2008.md]: It's not yet clear to me what "low-recycling" means in terms of groundwater recharge. Is recycling not just referring to reuse of treated municipal water?

File: _examples/groves_2008.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <p> Four workshops were organised, the first three designing the simulation model to evaluate the IEUA's water management plans over a wide range of futures, and the fourth  raising decision maker awareness about the significance of climate change risks for the actual plan relative to other alternatives and evaluating means to reduce the UWMP vulnerabilities to multiple plausible futures.</p>
+    <p> The project aimed to build decision makers' capacity to plan in the long-term and adapt to changing conditions. 
+    </p>
+    <p> The UWMP was designed based on a standard probability analysis approach to identify the best management option in terms of <a href="../../theory/reliability/">reliability</a> (i.e., chance of a water shortage under 7% over the next 25 years). The RDM approach identified two major vulnerable conditions for the plans:<ul>
```
Comment (josephguillaume) [inline _examples/groves_2008.md]: The specific example given here is focused on reliability, which is why I added the link. Is this ok, or did the UWMP go beyond this?

**Comment by Caroline-nc** [comment]
The idea of recycling water is associated to the risks of water shortage to water users for indoor and outdoor uses and associated increase in costs to access water, and to reduce pressure on water supplies.
The initial plan aims to 1) recycle water mainly for water users’ indoor and outdoor uses and to foster water conservation practices and 2) replenish aquifers, from SW runoff and other sources including recycled water.
I think the information under brackets could be removed, as not very helpful for understanding.

The two scenarios are based on whether or not the water supplies would not sustain the demands (Dry,…) or exceed (Wet,…) it, having implications in terms of costs due to shortages and water deliveries to users, on the one hand, and overinvestment in water supply enhancements, imposing unnecessary costs, respectively. There are not much explanations in the text regarding these scenarios except in terms of socioeconomic costs to water users.

I hope that would help.

**Comment by Caroline-nc** [comment]
No it is fine. I have checked the UWMP website too and the main objective is ‘reliability’.

**Pull request merged**